OrbitQt/orbitmainwindow.cpp

```diff
@@ -769,14 +769,17 @@ void OrbitMainWindow::RestoreDefaultTabLayout() {
 }
 
 void OrbitMainWindow::OnTimerSelectionChanged(const orbit_client_protos::TimerInfo* timer_info) {
-  if (!timer_info) return;
-  uint64_t function_address = timer_info->function_address();
-  LiveFunctionsDataView& live_functions_data_view =
-      ui->liveFunctions->GetLiveFunctionsController().GetDataView();
-  int selected_row = live_functions_data_view.GetRowFromFunctionAddress(function_address);
-  if (selected_row != -1) {
-    ui->liveFunctions->onRowSelected(selected_row);
+  std::optional<int> selected_row(std::nullopt);
+  if (timer_info) {
+    uint64_t function_address = timer_info->function_address();
+    LiveFunctionsDataView& live_functions_data_view =
+        ui->liveFunctions->GetLiveFunctionsController().GetDataView();
+    selected_row = live_functions_data_view.GetRowFromFunctionAddress(function_address);
```

Comment:
Why not also modify this function on DataView to return an optional?

Reply:
Good suggestion! Thanks!